flux-tests/tests/pos/surface/extern_spec_macro.rs

```diff
@@ -0,0 +1,16 @@
+// ignore-test
+// Need to add macros to test
+#![feature(register_tool)]
+#![register_tool(flux)]
+
+use std::mem::swap;
+
+#[extern_spec]
+#[flux::sig(fn(&mut i32[@a], &mut i32{v : a < v }) -> ())]
+fn swap(a: &mut i32, b: &mut i32);
+
+pub fn test() {
+  let mut x = 1;
+  let mut y = 2;
+  swap(&mut x, &mut y);
```

Comment:
You should probably enable format on save. I've seen you pushing commits just to fix formatting issues

```suggestion
    let mut x = 1;
    let mut y = 2;
    swap(&mut x, &mut y);
```

Reply:
Not sure it would fix this one since I wrote this test on the command line 😛